.github/workflows/turbo-daily-integration-test.yml

```diff
@@ -0,0 +1,67 @@
+# A workflow to run next.js integration test with turbopack for each day.
+# This runs against main branch with latest Next.js release.
+name: Daily Next.js integration test with turbopack
+
+on:
+  schedule:
+    - cron: '0 8 * * *'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Next.js version, sha, branch to test
+        type: string
+        default: 'canary'
+
+      post_to_slack:
+        description: Post test results to Slack
+        type: boolean
+        default: false
+
+jobs:
+  # Trigger actual next.js integration tests.
+  next_js_integration:
+    name: Execute Next.js integration workflow
+    permissions:
+      pull-requests: write
+    uses: ./.github/workflows/nextjs-integration-test.yml
```

Comment:
prolly as followup, this may able to use existing workflow in next.js repo itself - most of these setups are mimic to port what next.js already have.